autoreject/autoreject.py

```diff
@@ -191,12 +185,39 @@ def get_rejection_threshold(epochs):
         elif ch_type == 'grad':
             this_epoch = epochs.copy().pick_types(meg='grad', eeg=False)
 
-        _, test_scores = validation_curve(
-            GlobalAutoReject(), this_epoch, y=None,
-            param_name="thresh", param_range=param_range, cv=5)
+        X = this_epoch.get_data()
+        deltas = np.array([np.ptp(d, axis=1) for d in X])
+        all_threshes = np.sort(deltas.max(axis=1))
+        n_epochs, n_channels, n_times = X.shape
+
+        print('Estimating rejection dictionary for %s' % ch_type)
+        cache = dict()
+        est = GlobalAutoReject()
+        est.n_channels = n_channels
+        est.n_times = n_times
```

Comment:
est = GlobalAutoReject(n_channels=n_channels, n_times=n_times)

should work

Reply:
True ... it's cruft of copy paste. I'll fix